integration-tests/paho/src/test/java/org/apache/camel/quarkus/component/paho/it/PahoTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.camel.quarkus.component.paho.it;
+
+import java.util.UUID;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+@QuarkusTest
+@QuarkusTestResource(ActiveMQTestResource.class)
+class PahoTest {
+
+    @Test
+    public void test() {
+        final String msg = UUID.randomUUID().toString().replace("-", "");
+        RestAssured.given() //
+            .contentType(ContentType.TEXT).body(msg).post("/paho/publish") //
+            .then().statusCode(200);
```

Comment:
I do not find this test sufficient. I think there should be an assertion that the message really passed the queue. You could e.g. add another REST endpoint that would forward the messages from the queue using a ConsumerTemplate, something like we have here: https://github.com/apache/camel-quarkus/blob/master/integration-tests/twitter/src/main/java/org/apache/camel/quarkus/component/twitter/CamelResource.java#L58

Reply:
OK, I get your idea and will improve the test. Thanks @ppalaga